main/lsp/requests/hover.cc

```diff
@@ -55,7 +55,7 @@ unique_ptr<ResponseMessage> HoverTask::runRequest(LSPTypecheckerDelegate &typech
         auto level = fref.data(gs).strictLevel;
         if (level < core::StrictLevel::True) {
             auto text = fmt::format("This file is `# typed: {}`.\n"
-                                    "Hover, Go To Definition, and other features are disabled in this file.",
+                                    "Hover, Go To Definition, and other Sorbet-based features are disabled in this file.",
```

Comment:
Alternatively "Sorbet-powered" or "Sorbet-driven"?

Reply:
let's do Sorbet-powered, and also you will need to clang-format this file.

I can't do a multi-line suggestion outside the diff:

```cpp
        if (level < core::StrictLevel::True) {
            auto text =
                fmt::format("This file is `# typed: {}`.\n"
                            "Hover, Go To Definition, and other Sorbet-powered features are disabled in this file.",
                            level == core::StrictLevel::Ignore ? "ignore" : "false");
```

this should be what you need